src/.eslintrc.cjs

```diff
@@ -7,7 +7,4 @@ module.exports = {
       'peerDependencies': false,
     }],
   },
-  'env': {
-    'mocha': true,
-  },
```

Comment:
@hsablonniere I used `/* eslint-env node, mocha */` comment as you suggested, placed after the import statements in every test file :+1: